README.md

```diff
@@ -18,17 +18,24 @@ A curated list of amazingly awesome resources for (not only) LMC developers.
     - [Why write tests](https://www.ibuildings.nl/blog/2016/08/why-write-tests)
 - [SOLID, GRASP](https://www.zdrojak.cz/serialy/principy-objektove-orientovaneho-navrhu/)
 - [Verzování Databáze - Martin Major](https://www.youtube.com/watch?v=KTmlw5AKM8E)
+- [Technický dluh](http://blog.think-forth.com/2016/01/21/technicky-dluh/)
 
 ### PHP
 - [Symfony komponenty](https://www.zdrojak.cz/serialy/symfony-po-kruckach/)
+- [Fractal of bad design](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/)
+- [Formatting exceptions](http://rosstuck.com/formatting-exception-messages/)
+- [Doctrine hydratation optimization](https://ocramius.github.io/blog/doctrine-orm-optimization-hydration/)
+- [Functional programming](https://speakerdeck.com/pwm/functional-programming-in-php)
```

Comment:
Also a nice topic. But don't you think this is very "pro" one? I'm not sure its worth it to confuse newcommers and junior-devs with this concepts, and PHP is not functional language, so "FP in PHP" is by definition not an essential knowledge.

But I think we may add something more introductory to basic FP concepts. Don't you know any (short) interesting article about this?